diff --git a/Makefile b/Makefile
index 2214a94..4e340d4 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,9 @@
 
-PREFIX = /usr
+PREFIX ?= /usr
 DESTDIR =
 
+CXXFLAGS += `pkg-config --cflags lv2`
+
 all: build
 
 build:
diff --git a/Makefile.single b/Makefile.single
index ab3158e..ef8398a 100644
--- a/Makefile.single
+++ b/Makefile.single
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = __EFFECT__
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-AmpVTS.lv2/Makefile b/plugins/mod-caps-AmpVTS.lv2/Makefile
index 1c2519f..dae33ed 100644
--- a/plugins/mod-caps-AmpVTS.lv2/Makefile
+++ b/plugins/mod-caps-AmpVTS.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = AmpVTS
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-AutoFilter.lv2/Makefile b/plugins/mod-caps-AutoFilter.lv2/Makefile
index 1cd906f..03af1d7 100644
--- a/plugins/mod-caps-AutoFilter.lv2/Makefile
+++ b/plugins/mod-caps-AutoFilter.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = AutoFilter
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-CEO.lv2/Makefile b/plugins/mod-caps-CEO.lv2/Makefile
index 67f1c47..d0c5a54 100644
--- a/plugins/mod-caps-CEO.lv2/Makefile
+++ b/plugins/mod-caps-CEO.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = CEO
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-CabinetIII.lv2/Makefile b/plugins/mod-caps-CabinetIII.lv2/Makefile
index c99307b..09081ea 100644
--- a/plugins/mod-caps-CabinetIII.lv2/Makefile
+++ b/plugins/mod-caps-CabinetIII.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = CabinetIII
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-CabinetIV.lv2/Makefile b/plugins/mod-caps-CabinetIV.lv2/Makefile
index 71b3a21..e794482 100644
--- a/plugins/mod-caps-CabinetIV.lv2/Makefile
+++ b/plugins/mod-caps-CabinetIV.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = CabinetIV
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-ChorusI.lv2/Makefile b/plugins/mod-caps-ChorusI.lv2/Makefile
index 7b932c4..675a91c 100644
--- a/plugins/mod-caps-ChorusI.lv2/Makefile
+++ b/plugins/mod-caps-ChorusI.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = ChorusI
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Click.lv2/Makefile b/plugins/mod-caps-Click.lv2/Makefile
index b191e98..57284c1 100644
--- a/plugins/mod-caps-Click.lv2/Makefile
+++ b/plugins/mod-caps-Click.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Click
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Compress.lv2/Makefile b/plugins/mod-caps-Compress.lv2/Makefile
index 4d8f695..45cf05f 100644
--- a/plugins/mod-caps-Compress.lv2/Makefile
+++ b/plugins/mod-caps-Compress.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Compress
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-CompressX2.lv2/Makefile b/plugins/mod-caps-CompressX2.lv2/Makefile
index 9f3884c..96c09c8 100644
--- a/plugins/mod-caps-CompressX2.lv2/Makefile
+++ b/plugins/mod-caps-CompressX2.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = CompressX2
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Eq10.lv2/Makefile b/plugins/mod-caps-Eq10.lv2/Makefile
index c25f0c3..e6ae862 100644
--- a/plugins/mod-caps-Eq10.lv2/Makefile
+++ b/plugins/mod-caps-Eq10.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Eq10
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Eq10X2.lv2/Makefile b/plugins/mod-caps-Eq10X2.lv2/Makefile
index bfeb76b..169e7a5 100644
--- a/plugins/mod-caps-Eq10X2.lv2/Makefile
+++ b/plugins/mod-caps-Eq10X2.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Eq10X2
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Eq4p.lv2/Makefile b/plugins/mod-caps-Eq4p.lv2/Makefile
index 75ecd01..647a88a 100644
--- a/plugins/mod-caps-Eq4p.lv2/Makefile
+++ b/plugins/mod-caps-Eq4p.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Eq4p
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-EqFA4p.lv2/Makefile b/plugins/mod-caps-EqFA4p.lv2/Makefile
index ca8e610..75cb4e1 100644
--- a/plugins/mod-caps-EqFA4p.lv2/Makefile
+++ b/plugins/mod-caps-EqFA4p.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = EqFA4p
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Fractal.lv2/Makefile b/plugins/mod-caps-Fractal.lv2/Makefile
index 071062b..8aad5f8 100644
--- a/plugins/mod-caps-Fractal.lv2/Makefile
+++ b/plugins/mod-caps-Fractal.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Fractal
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Narrower.lv2/Makefile b/plugins/mod-caps-Narrower.lv2/Makefile
index 597f729..506b155 100644
--- a/plugins/mod-caps-Narrower.lv2/Makefile
+++ b/plugins/mod-caps-Narrower.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Narrower
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Noisegate.lv2/Makefile b/plugins/mod-caps-Noisegate.lv2/Makefile
index 13fa4ed..4266369 100644
--- a/plugins/mod-caps-Noisegate.lv2/Makefile
+++ b/plugins/mod-caps-Noisegate.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Noisegate
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-PhaserII.lv2/Makefile b/plugins/mod-caps-PhaserII.lv2/Makefile
index 3a02a2c..e170f1b 100644
--- a/plugins/mod-caps-PhaserII.lv2/Makefile
+++ b/plugins/mod-caps-PhaserII.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = PhaserII
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Plate.lv2/Makefile b/plugins/mod-caps-Plate.lv2/Makefile
index b52342b..9a4b137 100644
--- a/plugins/mod-caps-Plate.lv2/Makefile
+++ b/plugins/mod-caps-Plate.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Plate
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-PlateX2.lv2/Makefile b/plugins/mod-caps-PlateX2.lv2/Makefile
index 1e45137..adbc244 100644
--- a/plugins/mod-caps-PlateX2.lv2/Makefile
+++ b/plugins/mod-caps-PlateX2.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = PlateX2
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Saturate.lv2/Makefile b/plugins/mod-caps-Saturate.lv2/Makefile
index 113262a..6cdcd73 100644
--- a/plugins/mod-caps-Saturate.lv2/Makefile
+++ b/plugins/mod-caps-Saturate.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Saturate
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Scape.lv2/Makefile b/plugins/mod-caps-Scape.lv2/Makefile
index f2fd19d..10107f2 100644
--- a/plugins/mod-caps-Scape.lv2/Makefile
+++ b/plugins/mod-caps-Scape.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Scape
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Sin.lv2/Makefile b/plugins/mod-caps-Sin.lv2/Makefile
index 21724fa..4065619 100644
--- a/plugins/mod-caps-Sin.lv2/Makefile
+++ b/plugins/mod-caps-Sin.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Sin
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Spice.lv2/Makefile b/plugins/mod-caps-Spice.lv2/Makefile
index 2b2bbc4..486e8b7 100644
--- a/plugins/mod-caps-Spice.lv2/Makefile
+++ b/plugins/mod-caps-Spice.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Spice
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-SpiceX2.lv2/Makefile b/plugins/mod-caps-SpiceX2.lv2/Makefile
index a881751..6e993c9 100644
--- a/plugins/mod-caps-SpiceX2.lv2/Makefile
+++ b/plugins/mod-caps-SpiceX2.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = SpiceX2
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-ToneStack.lv2/Makefile b/plugins/mod-caps-ToneStack.lv2/Makefile
index 5cd3125..8aa1399 100644
--- a/plugins/mod-caps-ToneStack.lv2/Makefile
+++ b/plugins/mod-caps-ToneStack.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = ToneStack
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-White.lv2/Makefile b/plugins/mod-caps-White.lv2/Makefile
index 8c1ed8f..44c206b 100644
--- a/plugins/mod-caps-White.lv2/Makefile
+++ b/plugins/mod-caps-White.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = White
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
diff --git a/plugins/mod-caps-Wider.lv2/Makefile b/plugins/mod-caps-Wider.lv2/Makefile
index a1c7bbf..c263658 100644
--- a/plugins/mod-caps-Wider.lv2/Makefile
+++ b/plugins/mod-caps-Wider.lv2/Makefile
@@ -1,6 +1,6 @@
 VERSION = 0.9.26
 
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 
 CXX ?= g++
@@ -30,7 +30,7 @@ OBJECTS = $(SOURCES:.cc=.o)
 PLUG   = Wider
 BUNDLE = mod-caps-$(PLUG).lv2
 ifndef LV2_DEST
-LV2DEST = $(PREFIX)/lib/lv2/$(BUNDLE)
+LV2DEST = $(PREFIX)/lv2/$(BUNDLE)
 else
 LV2DEST = $(LV2_DEST)/$(BUNDLE)
 endif
